filterapi/filterconfig.go

```diff
@@ -82,12 +78,14 @@ type Config struct {
 	Schema VersionedAPISchema `json:"schema"`
 	// ModelNameHeaderKey is the header key to be populated with the model name by the filter.
 	ModelNameHeaderKey string `json:"modelNameHeaderKey"`
-	// SelectedBackendHeaderKey is the header key to be populated with the backend name by the filter
+	// SelectedRouteHeaderKey is the header key to be populated with the route name by the filter
 	// **after** the routing decision is made by the filter using Rules.
-	SelectedBackendHeaderKey string `json:"selectedBackendHeaderKey"`
+	SelectedRouteHeaderKey string `json:"selectedRouteHeaderKey"`
 	// Rules is the routing rules to be used by the filter to make the routing decision.
 	// Inside the routing rules, the header ModelNameHeaderKey may be used to make the routing decision.
 	Rules []RouteRule `json:"rules"`
+	// Backends is the list of backends to which the request should be routed to when the headers match.
+	Backends []*Backend `json:"backends"`
```

Comment:
Sorry, How we map the Backends with the headers now? Seems they're separated, no obvisous relationship between them.

Reply:
oh you are right. that's necessary for #588 right? we need to store the route->backends info somewhere in here. would you mind sending a patch?